### PR TITLE
Documentation for cartridge.roles.metrics added

### LINKED ISSUE #81: Cartridge role is not described in documentation

### PR DESCRIPTION
**Pull request by ZolotarevIgor-mail-ru**
Closes #81

### CODE REVIEW

**Comment by ZolotarevIgor-mail-ru** [comment]
@lenkis

File: README.md [hunk 1]
```diff
@@ -291,6 +291,57 @@ sum by (thread_name) (idelta(tnt_cpu_thread[$__interval]))
   / scalar(idelta(tnt_cpu_total[$__interval]) / tnt_cpu_count)
 ```
 
+## Cartridge role
+
+`cartridge.roles.metrics` is role for [tarantool/cartridge](https://github.com/tarantool/cartridge). It allows to use default metrics in cartridge app and manage them with config.
```
Comment (opomuc) [inline README.md:296]: `is a role`, `in a cartridge application`

File: README.md [hunk 2]
```diff
@@ -291,6 +291,57 @@ sum by (thread_name) (idelta(tnt_cpu_thread[$__interval]))
...
+
+### Usage
+
+1. Add `metrics` package to dependencies in the .rockspec file. Ensure you're using version 0.3.0 or higher.
```
Comment (opomuc) [inline README.md:300]: `Make sure that you are using`

File: README.md [hunk 3]
```diff
@@ -291,6 +291,57 @@ sum by (thread_name) (idelta(tnt_cpu_thread[$__interval]))
...
+    },
+})
+```
+3. After role initiation, default metrics will be enabled and global label 'alias' will be set. If you need to use any metrics package functionality, you may get it as a cartridge service and use it like a regular package after `require`:
+```lua
+local cartridge = require('cartridge')
+local metrics = cartridge.service_get('metrics')
+```
+
+4. To view metrics by API entrypoint, use configuration of the form:
```
Comment (opomuc) [inline README.md:326]: `To view metrics through API endpoint use the following configuration:`

File: README.md [hunk 4]
```diff
@@ -291,6 +291,57 @@ sum by (thread_name) (idelta(tnt_cpu_thread[$__interval]))
...
+```yaml
+metrics:
+  export:
+    - path: '/path_for_json_metrics'
+      format: 'json'
+    - path: '/path_for_prometheus_metrics'
+      format: 'prometheus'
+```
+You're able to add several entrypoints of one format by different paths, like this:
```
Comment (opomuc) [inline README.md:335]: `It is possible to add several endpoints with the same format but different paths, for example:`

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 [![Build Status](https://travis-ci.com/tarantool/metrics.svg?branch=master)](https://travis-ci.com/tarantool/metrics)
 # Metrics
 
-Metrics is a tool to collect, manipulate metrics timeseriess.  
+Metrics is a tool to collect, manipulate metrics timeseriess.
```
Comment (opomuc) [inline README.md:4]: `to collect and manipulate`, `time series`

File: README.md [hunk 6]
```diff
@@ -182,7 +182,7 @@ metrics = require('metrics')
   Returns Histogram object.
   * `name` Collector name (string). Must be unique.
   * `help` (optional) Help description (string).
-  * `buckets` Histogram buckets (array of positive sorted numbers). `INF` bucket is added automatically. Default is 
+  * `buckets` Histogram buckets (array of positive sorted numbers). `INF` bucket is added automatically. Default is
```
Comment (opomuc) [inline README.md:185]: `an array of sorted positive numbers`, `Infinity bucket (INF) is appended automatically`

File: README.md [hunk 7]
```diff
@@ -197,14 +197,14 @@ metrics = require('metrics')
...
-  of `histogram_obj`.  
+  of `histogram_obj`.
   For `observation` description see `counter_obj:collect()` section.
 
 #### Average
   Can be used only as HTTP statistics collector (described below) and cannot be built explicitly.
 
 #### `average_obj:collect()`
-  Returns list of two observations: 
+  Returns list of two observations:
```
Comment (opomuc) [inline README.md:207]: `Returns a list of two observations`

File: README.md [hunk 8]
```diff
@@ -291,6 +291,57 @@ sum by (thread_name) (idelta(tnt_cpu_thread[$__interval]))
...
+
+`cartridge.roles.metrics` is a role for [tarantool/cartridge](https://github.com/tarantool/cartridge). It allows to use default metrics in a cartridge application and manage them with config.
+
+### Usage
+
+1. Add `metrics` package to dependencies in the .rockspec file. Make sure that you are using version 0.3.0 or higher.
+```lua
+dependencies = {
+    ...
+    'metrics == 0.3.0-1',
```
Comment (olegrok) [inline README.md:304]: Few lines before you say "0.3.0 or higher" I think it could be changed to `'metrics >= 0.3.0-1'` to be a bit more consistent.

**Comment by vasiliy-t** [comment]
resolved by #88

**Pull request closed without merging**